internal/rbd/rbd_util.go

```diff
@@ -2002,3 +2006,16 @@ func (ri *rbdImage) addSnapshotScheduling(
 
 	return nil
 }
+
+// getCephClientLogFileName compiles the complete log file path based on inputs.
+func getCephClientLogFileName(id, logDir, prefix string) string {
```

Comment:
can you please add unit testing for this one and do we need to have prefix check here?

Reply:
@Madhu-1 wanted to make this function look simple and not return any error and then handle it exclusively from the caller, I'm worried that kills the purpose of making this simplified. So just in case if anyone passes prefix as nothing, use "ceph-" which is quite common for all ceph logs. Again that's just me!

Reply:
Looks like You are passing prefix as rbd-nbd always that's why the other question

Reply:
Please add a test to `internal/rbd/rbd_util_test.go` so that some of the expected results are documented and validated automatically.